contracts/test/strategies/curvePoolBooster.mainnet.fork-test.js

```diff
@@ -20,7 +20,7 @@ describe("ForkTest: CurvePoolBooster", function () {
 
```

Comment:
lets also add Arbitrum fork tests and verify the initial state works. Ideally, we would want to verify that we can execute the `closeCampaing` function. Though I am not sure how complex it is to setup preconditions for it

Reply:
Yep, I'll add tests there too.
However we can try to add a test for `closeCampaing`, but it will not really test the complete logic, as we dont have the bridging message tech. But better than nothing 👍

Reply:
Can we create the campaign on Arbitrum and then also try to close it on Arbitrum? Or does the creation of the campain need to originate from the Mainnet?

Reply:
In our case the campaign originate from Mainnet. 
However, even if we deployed it on Arbitrum, from what I've understand, they still need to bridge the message. 
Because the main entry to create a bribe is not directly on VotemarketV2 but using the CampaignRemoteManager contract, which will bridge message.